pkg/cmd/server/api/v1/types.go

```diff
@@ -881,3 +888,12 @@ type LDAPQuery struct {
 	// Filter is a valid LDAP search filter that retrieves all relevant entries from the LDAP server with the base DN
 	Filter string `json:"filter"`
 }
+
+type AdmissionConfig struct {
+	// PluginConfig allows specifying a configuration file per admission control plugin
+	PluginConfig map[string]string `json:"pluginConfig"`
```

Comment:
> I thought we were trying to protect a non-kubelet-container-pulling pull mechanism.. the existing kubernetes admission controller will already force pods (build or otherwise) to PullAlways, right?

Correct. My assumption is that we need another admission controller to set forcePull for all builds. @deads2k is suggesting we enable them by default, but make the default configuration have them perform no-ops.